articles/protocols/saml/adfs.md

```diff
@@ -4,202 +4,137 @@ toc: true
+4. Provide the location to where you want the certificate exported. Click **Next**.
+5. Verify that the settings for your certificate are correct and  click **Finish**.
 
-Switch over to the **Identifiers** tab, and populate the **Relying Party Identifier** with the **Entity ID** value. Be sure to click **Add** to add the identifier to your list.
+## Create a SAML connection with Auth0 as the service provider
 
-![](/media/articles/protocols/saml-adfs/saml23.png)
+1. Follow the tutorial on creating a SAML connection where [Auth0 acts as the service provider](/protocols/saml/saml-sp-generic). Where prompted, upload the signing certificate you exported from ADFS.
 
-Switch over to the **Endpoints** tab, and select the placeholder URL you provided earlier. Click **Edit...**.
+    The sign in and sign out URLs are usually in the form of `https://your.adfs.server/adfs/ls`.
 
-![](/media/articles/protocols/saml-adfs/saml24.png)
+2. Click **Save**. A page with instructions for creating a new Relying Party Trust in ADFS appears displaying the exact values required for your Auth0 account/connection. Make a note of these values. 
 
-Populate the **Trusted URL** with the **Post-back URL** value.
+    Here is an example: 
+    
+    | Parameter | Example Value |
+    | - | - |
+    | Post-back URL | `https://{yourAuth0accountdomain}/login/callback}/login/callback?connection={your new SAML connection}` |
+    | Entity ID | `urn:auth0:account:connection` |
 
-![](/media/articles/protocols/saml-adfs/saml25.png)
+## Edit the Relying Party Trust
 
-Click **OK**. Finally, click **Apply** and exit the Properties window.
+1. In the ADFS console, go to **ADFS > Relying Party Trusts** using the left-hand navigation pane. Select the Relying Party Trust you created earlier and click **Properties** (located on the right-hand navigation pane). 
+2. Select the **Identifiers** tab, and populate the **Relying Party Identifier** with the **Entity ID** value from the previous screen. Be sure to click **Add** to add the identifier to your list.
+3. Select the **Endpoints** tab, and select the placeholder URL you provided earlier. Click **Edit...**.
+4. Populate the **Trusted URL** with the **Post-back URL** value.
+5. Click **OK**. Finally, click **Apply** and exit the Properties window.
 
-### Signing Requests
+## Optional: Enable sign requests
 
-**Optionally**, if you want to sign your SAML requests to the ADFS server:
+Optionally, you can sign your SAML requests to the ADFS server.
 
-1. Go to the **Settings** page for your SAMLP Identity Provider in the Dashboard
+1. Go to the **Settings** page for your SAMLP Identity Provider in the Auth0 Dashboard.
 2. Enable **Sign Requests**.
 3. Just below the **Sign Requests** toggle is a link to download your certificate.
 4. Return to ADFS and load the downloaded certificate using the **Signatures** tab of the Relying Party properties dialog.
 
-### Map Your Claims
+## Optional: Map additional claims
```

Comment:
This step is mandatory, not optional. As part of the instructions, you can add a suggested mapping:

   | LDAP Attribute | Outgoing Claim |
    | - | - |
    | E-Mail-Addresses | E-Mail Address |
    | Display-Name | Name |
    | User-Principal-Name | Name ID |
    | Given-Name | Given Name |
    | Surname | Surname |

Also, make a note that the `NameID` outgoing claim should always be present to ensure correct session handling (later on there's a troubleshooting entry for logouts that mentions that this claim is required).

Reply:
Updated